snorkel/labeling/analysis.py

```diff
@@ -11,87 +11,123 @@
 Matrix = Union[np.ndarray, sparse.csr_matrix]
 
 
-############################################################
-# Label Matrix Diagnostics
-############################################################
 def _covered_data_points(L: Matrix) -> np.ndarray:
-    """Returns an indicator vector where ith element = 1 if x_i is labeled by at
-    least one LF."""
+    """Get indicator vector z where z_i = 1 if x_i is labeled by at least one LF."""
```

Comment:
Where is the full docstring here?  Should have input args, return args, and for this- since very user facing- an example

Reply:
It's a private method...

Reply:
Put this in the contributing guidelines